packages/jupytercad-extension/src/mainview.tsx

```diff
@@ -31,19 +31,18 @@ import {
   WorkerAction
 } from './types';
 import { FloatingAnnotation } from './annotation/view';
-import { throttle } from './tools';
+import { getCSSVariableColor, throttle } from './tools';
 import { Vector2 } from 'three';
 
 // Apply the BVH extension
 THREE.BufferGeometry.prototype.computeBoundsTree = computeBoundsTree;
 THREE.BufferGeometry.prototype.disposeBoundsTree = disposeBoundsTree;
 THREE.Mesh.prototype.raycast = acceleratedRaycast;
 
-const DARK_BG_COLOR = 'linear-gradient(rgb(0, 0, 42), rgb(82, 87, 110))';
-const LIGHT_BG_COLOR = 'radial-gradient(#efeded, #8f9091)';
+const DEFAULT_MESH_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-inverse-layout-color4'));
+const DEFAULT_EDGE_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-inverse-layout-color2'));
+const SELECTED_MESH_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-brand-color0'));
```

Comment:
Not sure about these, it's probably too expensive to evaluate everytime we need  colors. I'll change it to evaluate only when the theme changes.